google-interface/src/credentials/auth.js

```diff
@@ -29,49 +29,39 @@ exports.getOAuth2ClientFromLocalCredentials = function getOAuth2ClientFromLocalC
   oauthClientCredentialsPath = oauthClientCredentialsPath || process.env['OAUTH_CLIENT_CREDENTIALS_FILE']
   oauthUserTokenPath = oauthUserTokenPath || process.env['OAUTH_USER_TOKEN_FILE']
 
-  function resolver(resolve, reject) {
-    // Load client secrets from a local file.
-    fs.readFile(oauthClientCredentialsPath, (err, credentials) => {
-      if (err) {
-        err.message = 'Error loading client oauth credentials file: ' + err.message
-        return reject(err)
-      }
-      // Authorize a client with credentials
-      authorize(JSON.parse(credentials), oauthUserTokenPath, resolve, reject)
-    })
-  }
+  const  credentials = await fs.readFile(oauthClientCredentialsPath)
```

Comment:
`await` ? So this function must be made `async`

Reply:
Done.